deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/service-external.yaml

```diff
@@ -0,0 +1,18 @@
+# External facing assigner libp2p endpoint, used for peering.
+kind: Service
```

Comment:
LGTM 👍 Only needs to be added to its corresponding `kusomization` resources. Otherwise it will not be picked up.